buf/registry/module/v1beta1/tag_service.proto

```diff
@@ -120,6 +126,34 @@ message CreateTagsResponse {
   repeated Tag tags = 1 [(buf.validate.field).repeated.min_items = 1];
 }
 
+message MoveTagsRequest {
+  // An individual request to move a Tag.
+  message Value {
+    // The Module associated with the Tag.
+    ModuleRef module_ref = 1 [(buf.validate.field).required = true];
+    // The Tag name.
+    string name = 2 [
```

Comment:
```suggestion
    repeated string names = 2 [
```

We will send multiple tags for the same resourceref/commitref. Also, could you update this as well in the Create RPC?

Reply:
Yup, just did :)